src/app/clusters/access-control-server/access-control-server.cpp

```diff
@@ -41,14 +47,28 @@ using Entry          = AccessControl::Entry;
 using EntryListener  = AccessControl::EntryListener;
 using ExtensionEvent = Clusters::AccessControl::Events::AccessControlExtensionChanged::Type;
 
+#if CHIP_CONFIG_USE_ACCESS_RESTRICTIONS
+using ArlChangedEvent = Clusters::AccessControl::Events::AccessRestrictionEntryChanged::Type;
+using ArlReviewEvent  = Clusters::AccessControl::Events::FabricRestrictionReviewUpdate::Type;
+
+constexpr uint16_t kMaxInstructionStringLength = 512;
+
+constexpr uint16_t kMaxRedirectUrlStringLength = 256;
```

Comment:
constexpr size_t

Reply:
These were removed for now (not yet used).  Part of https://github.com/project-chip/connectedhomeip/issues/35165